File: 风乱/程序/文章格式转换/FormatChange.m
clear all
clc
text = fileread("Text.txt");
output = "";
i = 1;
while( i <= length(text))
    if text(i) == 13
        if flag == 0
            %flag = 1;
            output = output + " ";
            output = output + " ";
            output = output + text(i);
            output = output + text(i+1);
            i = i + 1;
        else
            i = i + 1;
        end
    else
        flag = 0;
        output = output + text(i);
    end
    i = i + 1;
end

fid = fopen("output.txt",'w');
fprintf(fid,"%s",output);
fclose(fid);

File: 风乱/程序/文章格式转换/FormatChange.py

if __name__ =='__main__':
    with open('E:\Ori\OriArticle\风乱\程序\文章格式转换\Text.txt', 'r') as f:
        text = f.read()
    
    output = ""
    flag = 0
    print(text)
    for char in text:
        if (char == '\n'):
            if (flag == 0):
                flag = 1
                output+="  "+char
        else:
            flag = 0
            output+=char

    with open('E:\Ori\OriArticle\风乱\程序\文章格式转换\Output.txt', 'w') as f:
        f.write(output)

